qbraid_algorithms/qft/qft.py

```diff
@@ -29,7 +42,7 @@
 from .qft_lib import QFTLibrary
 
 
-def load_program(num_qubits: int) -> QasmModule:
+def generate_program(num_qubits: int) -> QasmModule:
```

Comment:
The docstr is not formatted correctly - 

<img width="2340" height="422" alt="Image" src="https://github.com/user-attachments/assets/75674083-c46d-41b0-b4e4-c33ce17a99c1" />